index.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Taas Real Estate</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.2/font/bootstrap-icons.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  
+ <header>
+   <!--Header section-->
+    <nav class="topNav">
+        <a href="#"><i class="bi bi-facebook"></i></a>
+        <a href="#"><i class="bi bi-twitter"></i></a>
+        <a href="#">English</a>
+        <a href="#">My page</a>
+        <a href="#" class="signup">Logout</a>
+    </nav>
+    <!--Navigation mobile-->
+        <nav class="menu_mobile">
+                <div class="curtain">
+                    <a href="#" class="close">&times;</a>
+                    <div class="curtain-content">
+                            <a href="about.html">About</a>
+                            <a class="menu-link" href="#newsUpdate">updates</a>
+                            <a class="menu-link" href="#blog">Blog</a>
+                            <a class="menu-link" href="#">Report</a>
+                            <a class="menu-link" href="#">Contact Us</a>
+                    </div>
+                </div>
+
+                <i class="fa fa-bars open"></i>
+        </nav>
+    <!--Navigation desktop-->
+        <nav class="menu_desktop">
+          <div class="logo">
+            <img src="images/logo.png" alt="Logo-Image">
+            <a href="">Taas Homes</a>
+          </div>
+            <div class="nav_menu">
+              <a href="about.html">About</a>
+              <a class="menu-link" href="#newsUpdate">updates</a>
+              <a class="menu-link" href="#blog">Blog</a>
+              <a class="menu-link" href="#">Report</a>
+              <a class="menu-link" href="#">Contact Us</a>
+            </div>
+        </nav>
+</header>
+<!--Home section-->
+<section class="introduction">
+  <div class="content">
+    <h5>"Welcome to Taas Real Estate Paradise"</h5>
+  <h1>10 Units Left! Ready to move in 2&3 Beds</h1>
```

Comment:
Kindly change the h1 text color here to match the given design 😄 
![image](https://user-images.githubusercontent.com/87448628/181751832-5bd51f02-83bc-4685-884e-3b2972eecb7f.png)



![image](https://user-images.githubusercontent.com/87448628/181751662-96b66089-f774-408b-bf06-8c03fcbd1635.png)